colour/colorimetry/blackbody.py

```diff
@@ -594,3 +600,513 @@ def sd_blackbody(
         wavelengths,
         name=f"{temperature}K Blackbody",
     )
+
+
+def rayleigh_jeans_law(
+    wavelength,
+    temperature,
+) -> FloatingOrNDArray:
+    """
+    Returns the spectral radiance of electromagnetic radiation as a
+    function of wavelength from a black body at a given temperature
+    :math:`T[K]` in a medium having index of refraction :math:`n`.
+
+    Parameters
+    ----------
+    wavelength
+        Wavelength in meters.
+    temperature
+        Temperature :math:`T[K]` in kelvin degrees.
+
+    Returns
+    -------
+    :class:`numpy.floating` or :class:`numpy.ndarray`
+        Radiance in *watts per steradian per square metre* (:math:`W/sr/m^2`).
+
+    Notes
+    -----
+    -   The following form implementation is expressed in term of
+        wavelength.
+    -   The SI unit of radiance is *watts per steradian per square metre*
+        (:math:`W/sr/m^2`).
+
```

Comment:
We will need a `References` section here. You can use ``:cite:`Wikipedia2003f`` and the following top level reference:

```reStructuredText
-   :cite:`Wikipedia2003f` : Wikipedia. (2003). Rayleigh–Jeans law. Retrieved
    February 12, 2022, from https://en.wikipedia.org/wiki/Rayleigh–Jeans_law
```